svn/common.py

```diff
@@ -22,12 +22,32 @@ def __init__(self, message):
 
 
 class CommonClient(object):
+    """Subversion client
```

Comment:
I'm not a fan of the documentation-level commenting. It's not clear what documentation convention you are using and almost nothing else is has such comprehensive documentation-level commenting (ergo, it should be done everywhere, not just in one place).